target/step240-2020-1.0/WEB-INF/appengine-web.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To follow-up:
I believe no files under target/ should be submitted (see the .gitignore file).
Feel free to revert these changes in this PR and create a new PR that deletes the target/ directory.